app/services/subscriptions/create_service.rb

```diff
@@ -2,8 +2,6 @@
 
 module Subscriptions
   class CreateService < BaseService
-    include Utils::TransactionalJobs
```

Comment:
Really cool refactor! I think this was only used here so we can get rid of the concern definition too. If this PR takes a while for discussion (because I think it's a significant change) we could extract this part 🙏